projects/dotcms-field-elements/src/components/dot-select/dot-select.scss

```diff
@@ -0,0 +1,3 @@
+.dot-field__select--error {
+    border: 1px solid red;
```

Comment:
Are we doing this? I don't think is a good idea to add colors and/or any styles to the error state of the field.